static/app/components/replays/breadcrumbs/utils.tsx

```diff
@@ -3,12 +3,12 @@ import {BreadcrumbType, Crumb} from 'sentry/types/breadcrumbs';
 /**
  * Generate breadcrumb descriptions based on type
  */
-export function getDescription(crumb: Crumb, startTimestampMs?: number) {
+export function getDescription(crumb: Crumb) {
   if (typeof crumb.data === 'object' && crumb.data !== null && 'action' in crumb.data) {
     switch (crumb.data.action) {
       case 'largest-contentful-paint':
-        if (crumb.timestamp && startTimestampMs) {
-          return `${new Date(crumb.timestamp).getTime() - startTimestampMs}ms`;
+        if (crumb.data?.value !== undefined) {
+          return `${crumb.data.value}ms`;
```

Comment:
This means replays with older SDKs will not have any value, but only the first LCP value would be accurate. Wonder if we stick a SDK upgrade CTA here 😄

Reply:
nothing is better than something which is wrong imo.

if we're doing a cta to upgrade the sdk we should be looking at the most recent replay and it's version, because they might've just updated and this a replay from last week that's being viewed

Reply:
Good point re: most recent replay -- I think we'll make the CTA just a tooltip so it's less intrusive and keep it simple by looking at the replay's version.